### DISCUSSION TITLE
Is it possible to adjust the size of the Social Card description text?

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [squidfunk/mkdocs-material#3478] [Q&A] [answered]
My page descriptions are short. I'd like to change the size of the description text on Social Cards so they're easier to read. Is that possible?

The description text seems to break onto a new line while there is plenty of space left on the card. Is it possible to use more of the card width to avoid the description going over two lines? 

<img width="451" alt="twitter-card" src="https://user-images.githubusercontent.com/9379524/150119085-763348ee-95f1-4018-9c3f-719be244db86.png">

Thanks in advance 🙏

**Comment by Stephen-Gates** [comment, marked as answer]
Oh! Sorry, I just saw this is an enhancement request https://github.com/squidfunk/mkdocs-material/issues/2974 - any update?

**Reply by squidfunk** [reply]
Support for changing text sizes is not planned, because I don't think it's a good idea that users have to fiddle with font sizes for social cards. I'm more looking into the direction of auto-sizing text depending on the amount of text available, but it doesn't seem necessary at this moment. I understand that the rendering for your case is not perfect, but software like Material for MkDocs that is used by so many different projects with different requirements always has to do some trade-offs.

I'll see if we can improve the layout when I find some time to work on social cards again, but until then I wouldn't consider this an issue. If you wish to change the text yourself, [this is the line in question](https://github.com/squidfunk/mkdocs-material-insiders/blob/977e7218034a68b85e37c6ef053eeb0bb43cc64c/src/plugins/social/plugin.py#L158).

**Reply by Stephen-Gates** [reply]
Thanks for your quick reply. That seems a reasonable approach. 🙏
